### ISSUE TITLE
Learning rate not a member of org.deeplearning4j.nn.NeuralnetworkerConfigfuration

### ISSUE THREAD

**Issue by generaljam** [eclipse/deeplearning4j#8046]
#### Issue Description
Learning rate not a member of org.deeplearning4j.nn.NeuralnetworkerConfigfuration
Please describe our issue, along with:

I copied and pasted in my IntelliJ scala I throws me this error , and also for same error for different  hyper parameters , backprop
 -val multiLayerConf: MultiLayerConfiguration = new NeuralNetConfiguration.Builder()
  .seed(123).learningRate(0.1).iterations(1).optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT).updater(new Nesterovs(0.9)) //High Level Configuration
  .list() //For configuring MultiLayerNetwork we call the list method
  .layer(0, new DenseLayer.Builder().nIn(784).nOut(100).weightInit(WeightInit.XAVIER).activation(Activation.RELU).build()) //Configuring Layers
  .layer(1, new OutputLayer.Builder().nIn(100).nOut(10).weightInit(WeightInit.XAVIER).activation(Activation.SIGMOID).build())
  .pretrain(false).backprop(true) //Pretraining and Backprop Configuration
  .build() //Building Configuration
- encountered behavior
Error:(15, 20) value learningRate is not a member of org.deeplearning4j.nn.conf.NeuralNetConfiguration.Builder
.seed(123).learningRate(0.1).iterations(1).optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT).updater(new Nesterovs(0.9)) //High Level Configuration
#### Version Information
dl4j -version 1.0.0
nd4j-version 1.0.0 

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version
* Platform information (OS, etc)-
intelliJ ,OSx -mojave 

#### Additional Information

Where applicable, please also provide:

 <dependency>
            <groupId>org.deeplearning4j</groupId>
            <artifactId>deeplearning4j-core</artifactId>
            <version>1.0.0-beta4</version>
        </dependency>

        <dependency>
            <groupId>org.nd4j</groupId>
            <artifactId>nd4j-native-platform</artifactId>
            <version>1.0.0-beta4</version>
        </dependency>
#### Contributing

If you'd like to help us fix the issue by contributing some code, but would
like guidance or help in doing so, please mention it!

**Comment by raver119** [comment]
That's not a bug. Now learning rate is an argument of Updater.
